src/Runner.Common/StdoutTraceListener.cs

```diff
@@ -0,0 +1,27 @@
+using GitHub.Runner.Sdk;
+using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace GitHub.Runner.Common
+{
+    public sealed class StdoutTraceListener : RunnerTraceListener
+    {
+        // Console.Out is the stdout stream
+        public StdoutTraceListener() : base(Console.Out) { }
```

Comment:
this is exactly [ConsoleTraceListener](https://source.dot.net/System.Diagnostics.TextWriterTraceListener/R/d34b6cbcf7aebc48.html)

Reply:
Yes, I was inspired by https://github.com/microsoft/referencesource/blob/master/System/compmod/system/diagnostics/ConsoleTraceListener.cs

Reply:
what about leave `HostTraceListener.cs` unchanged and create
```
using System;
using System.Diagnostics;
using System.Globalization;
using GitHub.Runner.Sdk;

namespace GitHub.Runner.Common
{
    public sealed class RunnerStdoutTraceListener : ConsoleTraceListener
    {
        private readonly string _hostType;

        public RunnerStdoutTraceListener(string hostType)
        {
            _hostType = hostType;
        }

        // Copied and modified slightly from .Net Core source code. Modification was required to make it compile.
        // There must be some TraceFilter extension class that is missing in this source code.
        public override void TraceEvent(TraceEventCache eventCache, string source, TraceEventType eventType, int id, string message)
        {
            if (Filter != null && !Filter.ShouldTrace(eventCache, source, eventType, id, message, null, null, null))
            {
                return;
            }

            WriteHeader(source, eventType, id);
            WriteLine(message);
            WriteFooter(eventCache);
        }

        internal bool IsEnabled(TraceOptions opts)
        {
            return (opts & TraceOutputOptions) != 0;
        }

        // Altered from the original .Net Core implementation.
        private void WriteHeader(string source, TraceEventType eventType, int id)
        {
            string type = null;
            switch (eventType)
            {
                case TraceEventType.Critical:
                    type = "CRIT";
                    break;
                case TraceEventType.Error:
                    type = "ERR ";
                    break;
                case TraceEventType.Warning:
                    type = "WARN";
                    break;
                case TraceEventType.Information:
                    type = "INFO";
                    break;
                case TraceEventType.Verbose:
                    type = "VERB";
                    break;
                default:
                    type = eventType.ToString();
                    break;
            }

            Write(StringUtil.Format("[{0} {1:u} {2} {3}] ", _hostType, DateTime.UtcNow, type, source));
        }

        // Copied and modified slightly from .Net Core source code to make it compile. The original code
        // accesses a private indentLevel field. In this code it has been modified to use the getter/setter.
        private void WriteFooter(TraceEventCache eventCache)
        {
            if (eventCache == null)
                return;

            IndentLevel++;
            if (IsEnabled(TraceOptions.ProcessId))
                WriteLine("ProcessId=" + eventCache.ProcessId);

            if (IsEnabled(TraceOptions.ThreadId))
                WriteLine("ThreadId=" + eventCache.ThreadId);

            if (IsEnabled(TraceOptions.DateTime))
                WriteLine("DateTime=" + eventCache.DateTime.ToString("o", CultureInfo.InvariantCulture));

            if (IsEnabled(TraceOptions.Timestamp))
                WriteLine("Timestamp=" + eventCache.Timestamp);

            IndentLevel--;
        }
    }
}

```